FrontPage/auth/AuthViewController.swift

```diff
@@ -5,16 +5,12 @@ import AppAuth
 
 typealias PostRegistrationCallback = (_ configuration: OIDServiceConfiguration?, _ registrationResponse: OIDRegistrationResponse?) -> Void
 
-let authConfig = config.getConfiguration("keycloak").config ?? ["error": JSONValue.bool(false)]
-let server = authConfig["auth-server-url"]?.getString() ?? "error";
-let realms = authConfig["realm"]?.getString() ?? "error"
-
-let kIssuer: String = server + "/realms/" + realms + "/";
-let kClientID: String? = authConfig["resource"]?.getString() ?? "error";
-let kRedirectURI: String = "com.myapp://restore";
-let AuthStateKey: String = "authState";
+let kIssuer: String = Config.sharedInstance.getKIssuer()
+let kClientID: String? = Config.sharedInstance.getKClientId()
+let kRedirectURI: String = "com.myapp://restore"
```

Comment:
do we still need these to be global variables, or can we move them inside the class?

Reply:
got it moved. Thanks Wei!